server/protocols/jmap/src/test/java/org/apache/james/jmap/model/KeywordsTest.java

```diff
@@ -199,6 +199,7 @@ public void fromMapShouldNotThrowOnInvalidKeywordForLenientFactory() {
             .fromMap(ImmutableMap.of("in&valid", true)))
             .isEqualTo(Keywords.DEFAULT_VALUE);
     }
+
```

Comment:
Why is it in this commit?

Reply:
because checkstyle ask you to have one line between two methods